packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -161,7 +122,8 @@ const Statement = ({
             <ReportsMeta
                 className={is_mx_mlt ? undefined : 'reports__meta--statement'}
                 filter_component={filter_component}
-                optional_component={!is_switching && is_mx_mlt && getAccountStatistics()}
+                is_statement
+                optional_component={!is_switching && is_mx_mlt && AccountStatistics(account_statistics, currency)}
```

Comment:
Since `AccountStatistics` is a component, you should render it as a component as well